.chloggen/sdr-num.yaml

```
@@ -0,0 +1,30 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type:
+  bug_fix
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component:
+  redaction-processor
```

We typically use the full name like this:
```suggestion
component: redactionprocessor
```